### PR TITLE
[DSD-4272] Install.sh script for Credentialfeeder

### PR DESCRIPTION
**Pull request by VSIVAKALYAN**

### CODE REVIEW

File: deployment/v3/mosip/credential-feeder/install.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Installs idrepo
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=credentialfeeder
+CHART_VERSION=12.0.2
```
Comment (syedsalman3753) [inline deployment/v3/mosip/credential-feeder/install.sh:10]: In the helm chart, the chart version is set to 1.0.0. 

please get it confirmed by @ckm007

File: deployment/v3/mosip/credential-feeder/install.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+function installing_credentialfeeder() {
+  echo Istio label
+  kubectl label ns $NS istio-injection=enabled --overwrite
+  helm repo update
```
Comment (syedsalman3753) [inline deployment/v3/mosip/credential-feeder/install.sh:18]: helm repo add mosip is missing please add

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/v3/mosip/credential-feeder/install.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  helm repo add mosip https://mosip.github.io/mosip-helm
+
```
Comment (syedsalman3753) [inline deployment/v3/mosip/credential-feeder/install.sh:20]: helm repo update

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/v3/mosip/credential-feeder/install.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+NS=credentialfeeder
+CHART_VERSION=1.0.0
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+function installing_credentialfeeder() {
+  echo Istio label
+  kubectl label ns $NS istio-injection=enabled --overwrite
+  helm repo update
```
Comment (syedsalman3753) [inline deployment/v3/mosip/credential-feeder/install.sh:18]: please review this

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

**Review by syedsalman3753 — APPROVED** [review]

**Comment by ckm007** [comment]
please resolve coversations once it is fulfilled.

**Pull request merged**